c++/src/kj/common.h

```diff
@@ -1843,7 +1843,9 @@ class ArrayPtr: public DisallowConstCopyIfNotConst<T> {
     }
     return true;
   }
+#if !__cpp_impl_three_way_comparison
```

Comment:
TIL there are a lot of these macros: https://en.cppreference.com/w/cpp/feature_test